packages/core/src/codegen/pilet.codegen

```diff
@@ -1,4 +1,5 @@
 const { readFileSync } = require('fs');
+const { language } = require('../tools/meta');
```

Comment:
Should be taken from `meta-core`; `meta` contains additional info that is only available for the app shell (documentation main page).

Reply:
OK!